mmv1/third_party/terraform/tests/resource_workstations_workstation_config_test.go.erb

```diff
@@ -0,0 +1,94 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccorkstationsWorkstationConfig_update(t *testing.T) {
```

Comment:
Ah it's here! Just fix up the typo and it should work

Reply:
Fixed